knfc/knfconverter.js

```diff
@@ -0,0 +1,220 @@
+// Generated JS from Java: Node -----
+function Node() {
+    this.left = null;
+    this.right = null;
+    this.parent = null;
+    this.s = null;
+    this.id = null;
```

Comment:
In js definiert man glaube ich die Klassen Variablen nicht so, sondern man macht des Im Konstruktor.